paasta_tools/paasta_serviceinit.py

```diff
@@ -64,6 +65,8 @@ def main():
     else:
         log.setLevel(logging.WARNING)
 
+    configure_log()
```

Comment:
you mentioned now that this happens automatically, if so can we consistently *not* call this in main to tidy up the code a bit?